content/post/stash-v2022.05.18/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Introducing Stash v2022.05.12
+date: 2022-05-18
+weight: 10
+authors:
+  - Piyush Kanti Das
+tags:
+  - kubernetes
+  - stash
+  - backup
+  - restore
+  - docs
+  - nats
+  - elasticsearch
+---
+
+We are very excited to announce Stash `v2022.05.18`.  In this release, we have added exciting new features and improvements. We have squashed a few bugs as well. You can find the complete changelog [here](https://github.com/stashed/CHANGELOG/blob/master/releases/v2022.05.18/README.md). We are going to highlight the major changes in this post.
+
+## New Features
+
+Here, we are going to highlight the new features that have been introduced in this release.
+
+#### Add support for Elasticsearch 8.2.0
```

Comment:
- **Support for Elasticsearch 8.2.0:** We have added Elasticsearch 8.2.0 support to our Stash Elasticsearch add-on.

- **Support NATS Account Backup:** We have added Account Backup Support to our Stash NATS Addon. We have also upgraded the addon to support NATS version 2.8.1.